jsonschema-core.md

```diff
@@ -1856,62 +1848,52 @@ outcomes. However, these keywords are notable exceptions:
 
 The value of `unevaluatedItems` MUST be a valid JSON Schema.
 
-The behavior of this keyword depends on the annotation results of adjacent
-keywords that apply to the instance location being validated. Specifically, the
-annotations from `prefixItems`, `items`, and `contains`, which can come from
-those keywords when they are adjacent to the `unevaluatedItems` keyword. Those
-three annotations, as well as `unevaluatedItems`, can also result from any and
-all adjacent [in-place applicator](#in-place) keywords. This includes but is not
-limited to the in-place applicators defined in this document.
-
-If no relevant annotations are present, the `unevaluatedItems` subschema MUST be
-applied to all locations in the array. If a boolean true value is present from
-any of the relevant annotations, `unevaluatedItems` MUST be ignored. Otherwise,
-the subschema MUST be applied to any index greater than the largest annotation
-value for `prefixItems`, which does not appear in any annotation value for
-`contains`.
-
-This means that `prefixItems`, `items`, `contains`, and all in-place applicators
-MUST be evaluated before this keyword can be evaluated. Authors of extension
-keywords MUST NOT define an in-place applicator that would need to be evaluated
-after this keyword.
+This keyword applies to array instances by applying its subschema to the array's
+elements.
+
+The behavior of this keyword depends on all adjacent keywords as well as
+keywords in successfully validated subschemas that apply to the same instance
+location by evaluating the instance's elements. This includes, but is not
+limited to, `prefixItems`, `items`, and `contains`, itself, and all
+[in-place applicators](#in-place) defined in this document.
+
+This keyword applies its subschema to any array elements which have not been
+deemed "evaluated" per {{#unevaluated}}. Validation passes if the keyword's
```

Comment:
How is `{{#unevaluated}}` parsed/replaced here?

Reply:
It's replaced with the section title that's linked to the section.

Reply:
From the README ...

- [remark-reference-links](/json-schema-org/json-schema-spec/blob/main/remark-reference-link.js)
  -- Adds new syntax for referencing a section of the spec using the section number as the link text.
    - Example:
    ```markdown
    ## Foo {#foo}

    ## Bar
    This is covered in {{foo}} // --> Renders to "This is covered in [Section 2.3](#foo)"
    - Link text will use "Section" or "Appendix" as needed
    ```

So, it will determine the section number and generate the link text for the appropriate section. It's not necessarily the best practice to use numbers here, but that's how it was before we converted and we didn't change it. This functionality could easily be changed to use the section heading, but that's a bit off topic. Also off-topic, it would be nice to at some point have a build preview for reviewers.

Reply:
Looks like I do need to update this to `{{unevaluated}}` (without the `#`).